extensions/opentelemetry/opentelemetry/deployment/src/main/java/io/quarkus/opentelemetry/deployment/OpenTelemetryProcessor.java

```diff
@@ -43,6 +45,18 @@ public boolean getAsBoolean() {
         }
     }
 
+    static class RestClientAvailable implements BooleanSupplier {
+        @Override
+        public boolean getAsBoolean() {
+            try {
+                Class.forName("javax.ws.rs.client.ClientRequestFilter", false, Thread.currentThread().getContextClassLoader());
+                return true;
```

Comment:
See what I do in some of the MicrometerProcessors. You can do this once (static final) so you aren't rechecking every time the boolean supplier is used.

Reply:
Sure.